guestcluster/guest_cluster.go

```diff
@@ -0,0 +1,7 @@
+package guestcluster
+
+type GuestCluster struct {
+	ID        string `json:"id" yaml:"id"`
+	Namespace string `json:"namespace" yaml:"namespace"`
+	Service   string `json:"service" yaml:"service"`
```

Comment:
Godoc is coming later as soon as I know all this makes sense and works as intended.